integration-tests/react-percy-storybook/stories/index.js

```diff
@@ -86,3 +90,34 @@ storiesOf('Direction Demo', module).add('Show the direction', () =>
 storiesOf('Hierarchy.separator.is.supported', module).add('story', () =>
   <span>Hello hierarchySeparator</span>,
 );
+
+storiesOf('addWithPercyOptions', module)
+  .addWithPercyOptions('multiple widths', { widths: [222, 333] }, () =>
+    <span>Renders in multiple widths</span>,
+  )
+  .addWithPercyOptions('single width', { widths: [444] }, () => <span>Renders in one width</span>)
+  .addWithPercyOptions('without options', () => <span>Renders with the fallback width(s)</span>)
+  .addWithPercyOptions('with RTL of true for a single story', { rtl: true }, () =>
+    <div className={direction}>
+      <span>
+        The direction is {direction}.
+      </span>
+    </div>,
+  )
+  .addWithPercyOptions(
+    `${rtlRegex}: with RTL override of false even though the RTL regex matches`,
+    // rtl: false trumps a positive rtl_regex match
+    { rtl: false },
+    () =>
+      <span>
+        This story will only render in one direction. The direction is {direction} == ltr.
+      </span>,
```

Comment:
I've submitted a PR for a database migration for this.  I hope to have it run before Monday.